scripts/paths.js

```diff
@@ -1,22 +1,26 @@
-module.exports = {
-  "/thank-you": {
-    name: "thank-you",
-    path: "/thank-you/?",
-    handler: require('../pages/thank-you.jsx')
-  },
-  "/": {
-    name: "sequential",
-    path: "/?",
-    handler: require('../pages/sequential.jsx')
-  },
-  "/share": {
-    name: "share",
-    path: "/share/?",
-    handler: require('../pages/share.jsx')
-  },
-  "/give-bitcoin": {
-    name: "give-bitcoin",
-    path: "/give-bitcoin/?",
-    handler: require('../pages/give-bitcoin.jsx')
-  }
-};
+import currencies from './currencies.js';
+import locales from "../locales";
+import pages from "./pages.js";
+
+function getLocalePaths(arr) {
```

Comment:
Yeah, I like where you went with this.

I still kept them as separate function calls though. I wasn't sure where you were going with
```
getPaths((['/sequential-', '/thank-you-', '/paypal-donate-'], ['en', 'th', 'USD', 'CAD']);
```

We wouldn't want to end up with paths like '/sequential-cad'